controllers/volumereplicationgroup_controller.go

```diff
@@ -673,7 +659,7 @@ func (v *VRGInstance) separatePVCsUsingVRGStatus(pvcList *corev1.PersistentVolum
 		pvc := &pvcList.Items[idx]
 
 		for _, protectedPVC := range v.instance.Status.ProtectedPVCs {
-			if pvc.Name == protectedPVC.Name {
+			if pvc.Name == protectedPVC.Name && pvc.Namespace == protectedPVC.Namespace {
```

Comment:
This bit of code is common to volrep and volsync, but setting of namespace in volsync protected pvc was removed.  Either need to restore volsync code to set namespace or only require namespace match for volrep.

Reply:
Empty VRG.Status.ProtectedPVC namespace fields are set to VRG's namespace right after VRG's status is saved.